generators/app/index.js

```diff
@@ -185,6 +187,52 @@ module.exports = class extends BaseGenerator {
         }
 
         this.entities.forEach(entity => {
+            if (entity === 'no_entity') {
+                return;
+            }
+
+            this.entityClass = entity;
+            this.camelCaseEntityClass = _.camelCase(entity);
+
+            if (this.mustGenerateComponent(entity, 'consumer')) {
```

Comment:
This block could be refactor to avoid duplication with block in noEntitiesPrefix  
```javascript
 if (this.mustGenerateComponent(entity, 'consumer')) {
                this.template(
                    'src/main/java/package/service/kafka/consumer/EntityConsumer.java.ejs',
                    `${javaDir}service/kafka/consumer/${entity}Consumer.java`,
                    null,
                    null
                );
                this.template(
                    'src/main/java/package/service/kafka/deserializer/EntityDeserializer.java.ejs',
                    `${javaDir}service/kafka/deserializer/${entity}Deserializer.java`,
                    null,
                    null
                );
                this.template(
                    'src/main/java/package/service/kafka/deserializer/DeserializationError.java.ejs',
                    `${javaDir}service/kafka/deserializer/DeserializationError.java`,
                    null,
                    null
                );
            }

            if (this.mustGenerateComponent(entity, 'producer')) {
                this.template(
                    'src/main/java/package/service/kafka/producer/EntityProducer.java.ejs',
                    `${javaDir}service/kafka/producer/${entity}Producer.java`,
                    null,
                    null
                );
                this.template(
                    'src/main/java/package/service/kafka/serializer/EntitySerializer.java.ejs',
                    `${javaDir}service/kafka/serializer/${entity}Serializer.java`,
                    null,
                    null
                );
            }
```

Reply:
In fact I've started to do it but when it's done some vars like `packageName` in `.ejs` files are no longer retrieved because when extracting this code from `writing()` function to another the scope is not the same.

Reply:
Ok it's done in fact :)